client/style.css

```diff
@@ -0,0 +1,120 @@
+body {
+  display: flex;
+  flex-direction: column;
+  align-content: space-between;
+  margin: 0;
+}
+main {
+  text-align: center;
+}
+
+header {
+  display: flex;
+  height: 100px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+img {
+  width: 300px;
+  padding-top: 7rem;
+  margin-bottom: 2rem;
+}
+section {
+  display: flex;
+  flex-direction: column;
+}
+
+button {
+  border: 0px solid #5f6368;
+  background-color: #3f4347;
+  padding: 8px 10px;
+  border-radius: 5px;
+  color: white;
+  margin: 0 20px;
+}
+
+footer {
+  bottom: 0;
+  text-align: center;
+  background-color: #171717;
+  width: 100vw;
+  margin-top: 100px;
+  padding: 10px;
+}
+
+hr {
+  width: 100vw;
+}
+
+p {
+  margin: 10px 20px;
+  text-align: left;
+}
+
+a {
+  color: white;
+}
+.search-bar {
+  width: 450px;
+  border: 0;
+  outline: none;
+}
+
+.search-div:hover,
+.search-div:hover .search-bar,
+.search-div:focus-within,
+.search-bar:focus {
+  background-color: #5f6368;
+}
+
+.primary {
+  background-color: #202124;
+  color: white;
+}
+
+.search-div {
+  display: flex;
+  justify-content: center;
+  width: 500px;
+  border: 2px solid #5f6368;
+  height: 30px;
+  border-radius: 50px;
+  padding: 5px 0;
+  margin-bottom: 15px;
+}
+
+.form-div {
+  display: flex;
+  justify-content: center;
+}
+
+.links {
+  height: 100px;
+  padding: 10px;
+  align-content: center;
+}
+
+.img-google {
+  height: 30px;
+  width: 100px;
+  padding: 0 20px;
+  margin: 0;
+}
+
+.title-img {
+  align-content: center;
+}
+
+.inline {
+  display: inline;
+}
+
+.inline-search {
+  margin: 0;
+  padding: 0;
+}
+
+.grow {
+  min-height: 70vh;
+}
```

Comment:
Loved the styling of the home page, if you have time would be nice to include some styling for the results